Simplenote/src/main/java/com/automattic/simplenote/utils/NoteUtils.java

```diff
@@ -75,7 +75,12 @@ public void onClick(DialogInterface dialog, int whichButton) {
 
                             // Show empty placeholder for large devices in landscape.
                             if (activity instanceof NotesActivity) {
-                                ((NotesActivity) activity).showDetailPlaceholder();
+                                NotesActivity notesActivity = (NotesActivity) activity;
+                                if(notesActivity.getNoteListFragment() != null) {
+                                    notesActivity.getNoteListFragment().updateSelectionAfterTrashAction();
+                                } else {
+                                    ((NotesActivity) activity).showDetailPlaceholder();
```

Comment:
```suggestion
                                    notesActivity.showDetailPlaceholder();
```

Since the `notesActivity` variable is created above, we can use it in this statement, right?